pages/common/kdeconnect-cli.md

```diff
@@ -7,21 +7,21 @@
 
 `kdeconnect-cli --list-devices`
 
-- List all reachable devices:
+- List available (paired and reachable) devices:
 
 `kdeconnect-cli --list-available`
 
-- Request pairing with a specific device, specifying its ID:
+- Request pairing with a device, specifying its ID:
 
 `kdeconnect-cli --pair --device {{device_id}}`
 
-- Request pairing with a specific device, specifying its name:
+- Ring a device, specifying its name:
 
-`kdeconnect-cli --pair --name {{device_name}}`
+`kdeconnect-cli --ring --name {{device_name}}`
 
-- Ring a specific device:
+- Share a file or URL with a paired device, specifying its ID:
 
-`kdeconnect-cli --ring --name {{device_name}}`
+`kdeconnect-cli --share {{path/to/file|URL}} --device {{device_id}}`
```

Comment:
I'm thinking that putting `URL` before `path/to/file` might be less potentially ambiguous, but it's possible to read `path/to/file|URL` as "path to (file or URL)" — get it?
```suggestion
`kdeconnect-cli --share {{URL|path/to/file}} --device {{device_id}}`
```